### ISSUE TITLE
Create one command install&open script

### ISSUE THREAD

**Issue by bryk** [kubernetes/dashboard#1364]
We should create an one command install&open script for the Dashboard UI. On many repositories I see this and love the idea when I just want to check out a new thing. This could be like a banner on our front page:
`curl https://raw.github.com/kubernetes/dashboard/foo.sh | bash` 

It'd:
1. Open the browser to the UI if it is already installed.
2. Install the UI and go to point 1 otherwise.

**Comment by IanLewis** [comment]
Isn't the install currently just a kubectl one liner? What's the use case? is this for developers?

**Comment by bryk** [comment]
The use case is for users that want to try out the UI for the first time. The "paste this into your terminal" command can be called "install and launch" script. It'd basically the easiest way to have a live UI opened in your browser.

Use case: attract users by providing easiest possible launch procedure to them.

**Comment by floreks** [comment]
Should it be something that spins up the cluster, deploys UI and opens the browser or do we assume that user has running cluster already?

**Comment by bryk** [comment]
I'd assume that the user has kubectl configured first. It'd be much easier. We could use minikube to start a cluster in the v2 version. What do you think?

**Comment by cheld** [comment]
```shell
curl https://github.com/kubernetes/dashboard/blob/master/build/hyperkube.sh | bash
```

?

**Comment by bryk** [comment]
Yeah, something like this, but a bit more interactive and clever. I.e., open the UI when you have kubectl and UI already configured. I'll propose something over the weekend.
